templates/mathlib-overview.html

```diff
@@ -17,17 +17,17 @@ <h1>A mathlib overview</h1>
 	<p>
 	Here topics are listed in the greatest generality we currently have
 	in mathlib, hence some things may be difficult to recognize. We also
-	have a page dedicated to 
+	have a page dedicated to
 	<a href="undergrad.html">undergraduate mathematics</a> which may
 	be easier to read, as well as a page listing undergraduate maths topics
-	that are 
+	that are
 	<a href="undergrad_todo.html">not yet in mathlib</a>.
 	</p>
 
 	<p>
-	To make updates to this list, please make a pull request to this repository
-	after editing the 
-	<a href="https://github.com/leanprover-community/leanprover-community.github.io/blob/newsite/data/overview.yaml">yaml source file</a>.
+	To make updates to this list, please <a href="contribute/index.html">make a pull request to mathlib</a>
+	after editing the
+	<a href="https://github.com/leanprover-community/mathlib/blob/master/docs/overview.yaml">yaml source file</a>.
```

Comment:
I would also point out that PRs to yaml files can be done entirely online on GitHub (hopefully there is some GitHub help page we can link to).

Reply:
Good idea, thanks!